frontend/src/modules/userManagement/views/addUserView.js

```diff
@@ -5,36 +5,60 @@ define(function(require){
   var OriginView = require('core/views/originView');
 
   var AddUserView = OriginView.extend({
-    tagName: 'div',
     className: 'addUser',
     createdUserId: false,
 
+    events: {
+      'click .toggle-password': 'onTogglePassword'
+    },
+
     preRender: function() {
       Origin.trigger('location:title:update', { title: Origin.l10n.t('app.addusertitle') });
       this.listenTo(Origin, 'userManagement:saveUser', this.saveNewUser);
     },
 
     postRender: function() {
+      this.$password = this.$('[name="password"]');
       this.setViewToReady();
     },
 
     isValid: function() {
       var valid = true;
+      var map = Object.create(null);
 
       this.$('.field-error').each(function(index, element) {
         var $error = $(element);
         var $input = $error.siblings('input');
+        var values = {
+          $error: $error,
+          value: $input.val().trim()
```

Comment:
Causes the form to break since `$input` is no longer the right selector; there's now a `password-holder` `<div>` in there.